Running-Mastodon/Production-guide.md

```diff
@@ -111,7 +111,12 @@ server {
 
 ## Running in production without Docker
 
-It is recommended to create a special user for mastodon on the server (you could call the user `mastodon`), though remember to disable outside login for it. You should only be able to get into that user through `sudo -u mastodon`.
+It is recommended to create a special user for mastodon on the server (you could call the user `mastodon`), though remember to disable outside login for it. You should only be able to get into that user through `sudo su mastodon`.
+
+This command will create the user as needed:
+
+    sudo adduser --disabled-password --gecos "" mastodon
```

Comment:
using that requires changing home env var even when `--home dir` is used as `sudo -u <user> -s` copies the home var from the user who runs that.